tests/rules/no-invalid-properties.test.js

```diff
@@ -55,6 +55,37 @@ ruleTester.run("no-invalid-properties", rule, {
 		":root { --my-color: red; }\na { color: var(--my-color, var(--fallback-color, var(--foo, var(--bar)))) }",
 		":root { --my-color: red; }\na { color: var(--my-color, var(--fallback-color, var(--foo, var(--bar, blue)))) }",
 		":root { --color: red }\na { border-top: 1px var(--style, var(--fallback, solid)) var(--color, blue); }",
+		"a { width: calc(var(--my-width, 100px)) }",
```

Comment:
Can we add the following test case as well when the variable is also using calc():

```css
/* eslint css/no-invalid-properties: "error" */

:root { --dynamic-width: calc(20px + 10px); }
  
div { width: calc(100% - var(--dynamic-width)); }
```